recipe/meta.yaml

```diff
@@ -27,16 +27,16 @@ outputs:
         - param >=1.9.0
         # These are also needed at build time and declared in the pyproject.toml
         - pyct >=0.4.4
-        - bokeh >=2.0.0
+        - bokeh >=2.3.0,<2.4.0
         - nodejs >=10.13.0
         - wheel
-        - cartopy >=0.17.0
+        - cartopy >=0.18.0
         - setuptools >=30.3.0
       run:
         - python >=3.6
-        - bokeh >=2.0.0
-        - cartopy >=0.17.0
-        - holoviews >=1.13.0
+        - bokeh >=2.3.0,<2.4.0
```

Comment:
Are those upper bounds really necessary for bokeh ... we are at 2.3.1 on aggregate

Reply:
The Anaconda community innovation team maintains both bokeh and geoviews, and they warned us about this upcoming break. Having this constraint is a better option than having to patch repodata once the releases happen.